wpf/README.md

```diff
@@ -12,6 +12,9 @@ If you're new to .NET Core, here are a few resources to help you understand the
 ## Samples in this repo
 Coming soon!
 
+## Additional Samples
+Please visit [WPF Samples](https://www.github.com/Microsoft/wpf-samples) repo for additional WPF Samples that have been updated to target .NET Core 3.0. 
```

Comment:
two nits:
- Please visit --> See (we try to avoid the use of please in the documentation)
- additional WPF Samples --> additional WPF samples